src/app/shared/model/Feedback.model.ts

```diff
@@ -1,9 +1,9 @@
 export interface FeedbackModel {
```

Comment:
change feedback model data types to any method for handle API requests with any type of data